app/templates/components/preprint-navbar.hbs

```diff
@@ -9,7 +9,7 @@
         </div>
         <div id="preprint-navbar-links" class="navbar-collapse collapse navbar-right">
             <ul class="nav navbar-nav">
-                <li>{{#link-to "submit" class="" invokeAction=(action "click" "link" "Navbar - Add preprint")}}{{t "global.add_preprint"}}{{/link-to}}</li>
+                <li><a href='/preprints/submit' class="" invokeAction={{action "click" "link" "Navbar - Add preprint"}}>{{t "global.add_preprint"}}</a></li>
```

Comment:
This seemed to alleviate some of the weirdness caused by the ?edit query param. I am open to other suggestions.  Add/Edit are sharing same template and controller.  But Add is submit route and Edit is content route.  If "Add Preprint" in navbar is a link-to helper, and you follow these steps, you will see weird behavior. 1) Add a preprint as normal.  Press 'Share' at bottom of page and then confirm in modal. You will be taken to Content page 2) Press "edit" button at top of content page. You will be taken to Edit page 3) Press Add a Preprint button in navbar (button in question).  Complete form as normal.  Click share at bottom. You will be in Edit template instead of content page.

Also back button behavior was weird with this Add Preprint button.  You would get `http://localhost:5000/g5yw4?edit=true` and then hitting back again would give you the correct `http://localhost:5000/g5yw4/?edit`